documentation/staging/content/userguide/base-images/_index.md

```diff
@@ -131,24 +132,87 @@ to build the image and apply the patches.
+(for example, the data source password). For more information, see
+[WebLogic domain in container image protection]({{<relref "/security/domain-security/image-protection#weblogic-domain-in-container-image-protection">}}).
 {{% /notice %}}
+
+
+
+#### Apply patched images to a running domain
+
+When updating the WebLogic binaries of a running domain in Kubernetes with a patched container image,
+the operator applies the update in a Zero Downtime (ZDT) fashion. The procedure for the operator
+to update the running domain differs depending on the [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}). A
+difference between the domain home source types is the location of the domain home:
+
+* Domain in PV: The container image contains the JDK and WebLogic Server binaries. The domain home is located in a Persistent Volume (PV).
+* Model in Image: The container image contains the JDK, WebLogic Server binaries, and a WebLogic Deployment Tooling (WDT) model file and application archive file.
+* Domain in Image: The container image contains the JDK, WebLogic Server binaries, and domain home.   
+
+For Domain in PV, the operator can apply the update to the running domain without modifying the patched container image. For Model in Image (MII) and Domain in Image,
+before the operator can apply the update, the patched container images must be modified to add the domain home or a
+WDT model and archive. You use WebLogic Image Tool (WIT) [Rebase Image](https://github.com/oracle/weblogic-image-tool/blob/master/site/rebase-image.md)
+to update the Oracle Home of the original image using the patched Oracle Home from a patched container image.
+
+In all three domain home source types, you edit the [Domain Resource]({{< relref "/userguide/managing-domains/domain-resource#domain-resource-attribute-references" >}})
+to inform the operator of the name of the new patched image so that it can manage the update of the WebLogic domain.
+
+For a broader description of managing the evolution and mutation of container images to run WebLogic Server in Kubernetes, see [CI/CD]({{< relref "/userguide/cicd/_index.md" >}}).
+
+##### Domain in PV
+
+Edit the Domain Resource image reference with the new image name/tag (for example, `oracle/weblogic:12.2.1.4-patched`).
+Then, the operator performs a rolling restart of the WebLogic domain to update the Oracle Home of the servers.
+For information on server restarts, see [Restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}}).
+
+##### Model in Image
+Use the WIT [`rebase`](https://github.com/oracle/weblogic-image-tool/blob/master/site/rebase-image.md) command
+to update the Oracle Home for an existing image with the model and archive files in the image using the patched Oracle Home from a
+patched container image. Then, the operator performs a rolling update of the domain, updating the Oracle Home of each server pod.
```

Comment:
Consider linking to the life-cycle documentation for rolling update at each mention of rolling update.

Reply:
Will do.

Reply:
There is so much detailed information about rolling restarts, I'm tempted to cross-reference the Overview section, where it all begins, https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/restarting/#overview
Do you have a better idea?

Reply:
Actually, there is this cross-reference also, which might be a better one for rolling restarts, http://localhost:1313/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#rolling-restarts. I think I will use it.

Reply:
I think #restarting-servers is a better link than #rolling-restarts (restarting servers discusses what happens when you change the image, rolling-restarts goes into detail about how restarts are handled at the cluster level).

Reply:
Agree. I'll change them back.